<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Piscan</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="scan_tab">
       <attribute name="title">
        <string>Scan</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="GLScanWidget" name="GLScanWidgetui" native="true"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonProcessScan">
          <property name="text">
           <string>Process Scan</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings_tab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Camera HFOV</string>
            </property>
            <property name="buddy">
             <cstring>lineEditHFOV</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Camera VFOV</string>
            </property>
            <property name="buddy">
             <cstring>lineEditVFOV</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Camera Distance</string>
            </property>
            <property name="buddy">
             <cstring>lineEditDistance</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Laser Offset</string>
            </property>
            <property name="buddy">
             <cstring>lineEditLaserOffset</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Horizontal Average</string>
            </property>
            <property name="buddy">
             <cstring>lineEditHorizontalAverage</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Vertical Average</string>
            </property>
            <property name="buddy">
             <cstring>lineEditVerticalAverage</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Frame Skip</string>
            </property>
            <property name="buddy">
             <cstring>lineEditFrameSkip</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Line Skip</string>
            </property>
            <property name="buddy">
             <cstring>lineEditLineSkip</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="lineEditHFOV"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditVFOV"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditDistance"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditLaserOffset"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditHorizontalAverage"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditVerticalAverage"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditFrameSkip"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditLineSkip"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>304</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonResetSettings">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSaveSettings">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLScanWidget</class>
   <extends>QWidget</extends>
   <header>glscanwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditHFOV</tabstop>
  <tabstop>lineEditVFOV</tabstop>
  <tabstop>lineEditDistance</tabstop>
  <tabstop>lineEditLaserOffset</tabstop>
  <tabstop>lineEditHorizontalAverage</tabstop>
  <tabstop>lineEditVerticalAverage</tabstop>
  <tabstop>lineEditFrameSkip</tabstop>
  <tabstop>lineEditLineSkip</tabstop>
  <tabstop>pushButtonSaveSettings</tabstop>
  <tabstop>pushButtonResetSettings</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
